src/app/study/components/subject-table/subject-table.component.ts

```diff
@@ -25,12 +25,15 @@ export class SubjectTableComponent implements OnInit, OnDestroy {
   displayedColumns = [
     'subjectId',
     'humanReadableId',
-    'status',
+    'subjectStatus',
+    'deviceStatus',
     'sources',
     'lastSeen'
   ]
   dataSource: SubjectDataSource | null
   subjectDB = new SubjectDB()
+  statusCon = 'CONNECTED'
```

Comment:
what do we use these for? Aren't the status values read from response received?

Reply:
Yeah I initially used this for testing out the component. I'll remove this.